shell/utils/validators/machine-pool.ts

```diff
@@ -11,7 +13,15 @@ const RULESETS = [
   {
     path:  FIELDS.NAME,
     rules: ['required'],
-  }
+  },
+  {
+    path:  FIELDS.AUTOSCALER_MIN,
+    rules: ['requiredInt', 'isPositive', 'isAutoscalerMaxGreaterThanMin'],
```

Comment:
I put the isAutoscalerMaxGreaterThanMin in both min and max fields because these don't get evaluated until a user focuses on a field and I wanted to ensure that whichever field is touched the user will be informed.